src/DataFixtures/ORM/LoadEmailSubscriptionHistoryData.php

```diff
@@ -76,27 +76,22 @@ public function load(ObjectManager $manager)
         $manager->flush();
     }
 
-    private function createSubscribedUnsubscribedHistory(
-        Adherent $adherent,
-        array $tags,
-        string $subscribedAt,
-        string $unsubscribedAt = '-1 month',
```

Comment:
This argument is still passed to the method https://github.com/EnMarche/en-marche.fr/pull/2546/files#diff-c8b7e31a004d47136afcbde7cb6b1e82R61, but no more taken into account.

Reply:
@illusionOfParadise, the 4-th and 5-th arguments are never passed, that is why I remove them